src/components/Button/Button.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  children: React.ReactNode;
+  name: string;
+}
+
+const Button: React.FC<ButtonProps> = ({ children, name, ...props }) => {
+  return (
+    <>
```

Comment:
We can remove this extra empty `<>` fragment since we are only returning a single JSX element.